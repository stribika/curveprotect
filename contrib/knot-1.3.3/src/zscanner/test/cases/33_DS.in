$ORIGIN	.
$TTL	1

; OK
@	DS	0	0	0	00		; The simplest variant
@	DS	65535	255	255	00		; Maximal numbers
@	DS	0	RSAMD5	0	00		; Algorithm mnemonic
@	DS	0	0	0	01 02 0304	; Hex block with blank spaces between them
@	TYPE43	\# 5 0000000000				; TYPE + Hexadecimal rdata
@	TYPE43	0	0	0	00		; TYPE
@	ds	0	0	0	00		; Type in lower-case

; KO
@	DS
@	DS						; Empty rdata
@	DS	\# 0					; Hex empty rdata
@	DS	\#					; Missing hex length
@	DS	65536	0	0	00		; Key tag overflow
@	DS	0	256	0	00		; Algorithm overflow
@	DS	0	0	256	00		; Digest type overflow
@	DS	0	0	0	0		; Continuous block length must be multiple of 2
@	DS	0	0	0	00 0		; Continuous block length must be multiple of 2
@	DS	0	0	0	XX		; Bad hex character
@	DS	0	0	0			; Missing item
@	DS	\# 5 000000000000			; Too long rdata
@	DS	\# 6 0000000000				; Bad rdata length
