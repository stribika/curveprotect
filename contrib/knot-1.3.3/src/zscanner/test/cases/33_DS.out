OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002B
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002B
RDATA=FFFFFFFF00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002B
RDATA=0000010000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002B
RDATA=0000000001020304
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002B
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002B
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002B
RDATA=0000000000
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER8_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER8_OVERFLOW
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
